src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -1073,6 +1073,23 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <!--
+  ============================================================
+  Command Line Confirugation Validation
+  ============================================================
+  -->
+
+  <Target Name="ValidateCommandLineProperties"
+          AfterTargets="PreBuildEvent" >
```

Comment:
For a lot of similar checks, we do `BeforeTargets="_CheckForInvalidConfigurationAndPlatform"`.  Is there any reason this is different?